.ebextensions/yarn.config

```diff
@@ -0,0 +1,8 @@
+packages:
+  rpm:
+    nodesource: https://rpm.nodesource.com/pub_10.x/el/7/x86_64/nodesource-release-el7-1.noarch.rpm
+container_commands:
+  01_upgrade_nodejs:
+    command: yum -y install nodejs
```

Comment:
did yum package not work? Says on https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/customize-containers-ec2.html rpm should get done first